2-Draft-Accepted/RFCxxxx-PowerShellGet-3.0.md

```diff
@@ -132,16 +141,22 @@ A `-Trusted` switch indicates whether to prompt the user when installing resourc
 from that repository.
 By default, if this switch is not specified, the repository is untrusted.
 A `-Repositories` parameter will accept an array of hashtables equivalent to
-the parameter names.
+the parameter names (like splatting).
+
+```powershell
+Register-PSResourceRepository -Repositories @(
+  @{ URL = "https://one.com"; Trusted = $true }
+  @{ URL = "https://powershellgallery.com"; Trusted = $true; Default = $true }
```

Comment:
Are we giving up on friendly names for repositories?

Reply:
Yes, will do.

Reply:
@SteveL-MSFT please be sure to add an example of a local file system based repository. I haven't seen any example of those. It's a very important scenario, as it enables local validation without relying on an off-box service